### ISSUE TITLE
The value of "tau"  affects the number of ROI detected

### ISSUE THREAD

**Issue by Jessica0218** [MouseLand/suite2p#837]
Thank you very much for your work. I’m a beginner in calcium imaging analysis.

I'm recording in awake mice  using GCaMP6f. The "tau" value  is set to 0.7 as you recomend in your wiki. But the deconvolved data output from Suite2p is not good. So I change the “tau” to 2s , finding that the number of ROI detected increased significantly. Does this mean that tau is set to 2 more suitable for my data? Why does tau affect the number of ROI detected?  In my opinion, tau only affects spike detection (deconvolution).

**Comment by carsen-stringer** [comment]
ah yes, the tau determines how much the data is binned before ROI detection -- the bin size in time is `ops['tau']*ops['fs']`. this video has more info on ROI detection: https://youtu.be/NcC0YxQ9o3A

**Comment by carsen-stringer** [comment]
for noisy-ish data, more binning will help with ROI detection. we could add a separate parameter for this if you think it would be useful

**Comment by Z-Ghasemahmad** [comment]
@carsen-stringer Hi Carsen, I want to follow up on this question as I have noticed multiple issues arising from choice of tau value. 
We have recently transitioned to imaging using GCaMP8s. Previously using GCaMP6f, we used a tau value of 0.7 per recommendation. Per 8s dynamics , I changed the tau value and tested 0.1 and 0.2. There are significant differences in ROI detection and depending on the imaging session, the Ca2+ transients are also affected. My question is that, what value should I use? Shouldn't tau value be used for properly extracting Ca transient for the relevant GCaMP sensor? If so, how to fix the issue of ROI detection? If then try to use a tau value that results in best ROI detection, can it properly reflect the calcium transients and the cell activity in response to certain stimulus?

![Image](https://github.com/user-attachments/assets/f8e0c21a-e416-409f-9115-7b9d234972fb)
